packages/zlibpkg.rb

```diff
@@ -34,15 +34,20 @@ def self.patch
   end
 
   def self.build
-    system "cmake -B builddir -G Ninja #{CREW_CMAKE_OPTIONS}"
-    system 'samu -C builddir'
+    system "cmake \
+      -B builddir -G Ninja \
+      #{CREW_CMAKE_OPTIONS.gsub('-mfpu=vfpv3-d16', '-mfpu=neon-fp16')} \
+      -Wno-dev"
+    system "#{CREW_NINJA} -C builddir"
   end
 
   def self.check
-    system 'samu -C builddir test'
+    system "#{CREW_NINJA} -C builddir test"
   end
 
   def self.install
-    system "DESTDIR=#{CREW_DEST_DIR} samu -C builddir install"
+    system "DESTDIR=#{CREW_DEST_DIR} #{CREW_NINJA} -C builddir install"
+    # Remove static library.
+    FileUtils.rm "#{CREW_DEST_LIB_PREFIX}/libz.a"
```

Comment:
Is there a `-Dsomething` option to disable static?

Reply:
Not that I could see.

Reply:
(I looked.)

Reply:
I just checked and there is no cmake option.

Reply:
Bummer and lame.